CorsixTH/com.corsixth.corsixth.metainfo.xml

```diff
@@ -23,6 +23,9 @@
   <provides>
     <binary>corsix-th</binary>
   </provides>
+  <launchable>
+    <desktop-id>com.corsixth.corsixth</desktop-id>
```

Comment:
My understanding is that desktop-id is suppose to be the value of the type attribute of the launchable tag: https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-launchable

Reply:
Also .desktop should be part of the value